Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ however, it has to be formatted properly to pass verification tests.
 
 ## [1.6.2] - 2023-07-10
 
+### Changed
+- Changed the `InputAction` constructors so it generates an ID for the action and the optional binding parameter. This is intended to improve the serialization of input actions on behaviors when created through API when the property drawer in the Inspector window does not have a chance to generate an ID.
+
+### Fixed
+- Fixed serialization migration in the Tracked Pose Driver component causing bindings to clear when prefabs are used in some cases ([case ISXB-512](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-512), [case ISXB-521](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-521)).
+- Fixed the `Clone` methods of `InputAction` and `InputActionMap` so it copies the Initial State Check flag (`InputAction.wantsInitialStateCheck`) of input actions.
+
```

Comment:
I think we missed the window on 1.6.2, but I'll talk to Lyndon about fast-tracking another patch release to fix this.

Reply:
It was a bad merge, I fixed it by moving it from 1.6.2 to Unreleased.